### PR TITLE
Create Database Framework

### PR DESCRIPTION
**Pull request by SilverStrings024**
Created database framework to replace user_sids.json to prevent the need for sudo
NOTE: It's likely that, if installed globally, the program will need sudo to create the database..
TODO: Create dedicated script to create database that must be run as sudo once (unless package is uninstalled or removed).

### CODE REVIEW

File: pytextnow/database/db.py [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+        # Using a filter with a trailing ", " causes an error
+        # therefore it's unsafe until that's stripped off
+        unsafe_filters = ""
+        for column, value in filters.items():
+            unsafe_filters += f"{column}={value}, "
+        safe_filters = self.__clean_query(unsafe_filters)
+        sql = f"FROM {table_name} SELECT * WHERE "+safe_filters+";"
+        return self.__execute_sql(sql)
+
+    def __update_obj(self, table_name: str, new_data: typing.Dict[str, str]) -> None:
```
Comment (SilverStrings024) [inline pytextnow/database/db.py:149]: Use ` -> typing.Union[None, Results]` instead of `-> None`

**Review by leogomezz4t — CHANGES_REQUESTED** [review]
1. Delete the README.md from the pull requests because the one already in 1.1.8-test is ahead of this one.
2. You can delete the if __name__ == "__main__" import in TNAPI.py. It was just for debugging.
3. Remove 'self' from all Container objects instantiations.

**Comment by SilverStrings024** [comment]
>     1. Delete the README.md from the pull requests because the one already in 1.1.8-test is ahead of this one.
> 
>     2. You can delete the if **name** == "**main**" import in TNAPI.py. It was just for debugging.
> 
>     3. Remove 'self' from all Container objects instantiations.

I thought you said you took out all those references?

**Comment by leogomezz4t** [comment]
> > ```
> > 1. Delete the README.md from the pull requests because the one already in 1.1.8-test is ahead of this one.
> > 
> > 2. You can delete the if **name** == "**main**" import in TNAPI.py. It was just for debugging.
> > 
> > 3. Remove 'self' from all Container objects instantiations.
> > ```
> 
> I thought you said you took out all those references?

I did but your commit had some old code

**Comment by leogomezz4t** [comment]
Doesn't matter I already fixed TNAPI.py

**Review by bryanperez43 — COMMENTED** [review]
noticed user_sid.json had data.. deleted it

**Comment by leogomezz4t** [comment]
> noticed user_sid.json had data.. deleted it

thanks

**Comment by leogomezz4t** [comment]
I guess now we have to test it

**Review by leogomezz4t — APPROVED** [review]

**Comment by bryanperez43** [comment]
we need 

1. integrate the database
2. test / fix any bugs (if any)
3. re-write example scripts
4. edit the README.md examples

**Comment by SilverStrings024** [comment]
Don't forget also we have things that need tested listed in the integration channel on the discord. We still need to figure out who is doing what so we don't work on the same stuff.

**Comment by SilverStrings024** [comment]
Merging

**Comment by leogomezz4t** [comment]
ok

**Pull request merged**